FactoryTest/FactoryTest.ino

```diff
@@ -143,16 +177,16 @@ void writeNumberToLights(uint16_t number) {
     exerciseEachLED(0, numLEDs - 1, 50);
 
     writeHexDigitToLights((number >> 12) & 0xf);
-    delay(DELAY_500ms);
+    delay(DELAY_1Sec);
```

Comment:
1S looks like 15 and the ms is lowercase.... I'd downcase the s for redability or add another _

Reply:
Done.

Reply:
Done.